### PR TITLE
Update text to include enable check time

### PR DESCRIPTION
**Pull request by jzampieron**
## Description
Add language discussing use of ENABLE_CHECK_TIME and other time sync mechanisms.

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium
- [X] Low

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by joerg84** [comment]
@vishnu2kmohan could you take another looks?

File: 1.9/installing/custom/system-requirements/index.md [hunk 1]
```diff
@@ -131,7 +131,25 @@ Add the following line to your `/etc/sudoers` file. This disables the sudo passw
 
 Alternatively, you can SSH as the root user.
 
-### Enable NTP
+### Enable Time synchronization
+
+Time synchronization is a core requirement of DC/OS. There are various methods
+of ensuring time sync. NTP is the typical approach on bare-metal.
+Many cloud providers use hypervisors which push time
```
Comment (sascala) [inline 1.9/installing/custom/system-requirements/index.md]: comma after `hypervisors`

File: 1.9/installing/custom/system-requirements/index.md [hunk 2]
```diff
@@ -131,7 +131,25 @@ Add the following line to your `/etc/sudoers` file. This disables the sudo passw
...
+down to the VM guest operating systems. In certain circumstances hypervisor
```
Comment (sascala) [inline 1.9/installing/custom/system-requirements/index.md]: comma after `circumstances`

File: 1.9/installing/custom/system-requirements/index.md [hunk 3]
```diff
@@ -131,7 +131,25 @@ Add the following line to your `/etc/sudoers` file. This disables the sudo passw
...
+time-sync may conflict with NTP.
+
+You must understand how to properly configure time synchronization for your
+environment. When in doubt, enable NTP and check using `/opt/mesosphere/bin/check-time`.
+
+#### Enable Check Time
+
+There is an `ENABLE_CHECK_TIME` environment variable that needs to be set
```
Comment (sascala) [inline 1.9/installing/custom/system-requirements/index.md]: You must set the `ENABLE_CHECK_TIME` environment variable in order for `/opt/mesosphere/bin/check-time` to function. We recommend you enable this globally....

**Comment by joerg84** [comment]
@sascala Is this ok to be merged?

**Pull request merged**